index.html

```diff
@@ -62,21 +68,27 @@ <h3 class="ability_header">Skills</h3>
                     </ul>
                 </div>
     
-            </div>            
-            <ul class="social_media_links">
-                <li><i class="fa-brands fa-github"></i></li>
-                <li><i class="fa-brands fa-linkedin"></i></li>
-                <li><i class="fa-brands fa-twitter"></i></li>
-                <li><i class="fa-brands fa-angellist"></i></li>
-            </ul>
-            <div class="arrow_down">
-                <p><i class="fa-solid fa-arrow-down"></i>  Work</p>
-            </div>
+            </div>    
+            <div class="headline_base">
+                <div class="arrow_down arrow_mobile">
+                    <p><i class="fa-solid fa-arrow-down"></i>  Work</p>
+                </div>
+                <ul class="social_media_links">
+                    <li><i class="fa-brands fa-github"></i></li>
+                    <li><i class="fa-brands fa-linkedin"></i></li>
+                    <li><i class="fa-brands fa-twitter"></i></li>
+                    <li><i class="fa-brands fa-angellist"></i></li>
```

Comment:
[OPTIONAL] It would be better to wrap these icons in `a` tags and link them to your social profiles 😀
```
<li><a href="your-girhub-profile-url-here"><i class="fa-brands fa-github"></i></a></li>
```